packages/components/atoms/f-button/test-utils/component-objects/f-button.component.js

```diff
@@ -8,7 +8,7 @@ module.exports = class Buttons extends Page {
     open (buttonType = '') {
         const url = buttonType === 'link' ? '&knob-Button%20Type=link&knob-href=link' : '';
 
-        super.openComponent('atom', `button-component${url}`);
+        browser.url(`/iframe.html?id=components-atoms-f-button--button-component${url}`)
```

Comment:
Is there a way of making this work with the class method?

Maybe `super.openComponent('atom', `f-button-button-component${url}`);

Reply:
The changes should be done in page object model to make it work as I made f-button story to be a folder and not just a page, but @LTurns  told that they are refactoring the page object at the moment and recommended to do this  just by calling `browser.url` directly

Reply:
yeah exactly, Jonkers is working on refactoring the page object model with the components at the moment, so it saves the hassle of having to make these extra changes when it's going to be restructured 😄